### ISSUE TITLE
Why do you use "Linear" for beta schedule?

### ISSUE THREAD

**Issue by bonlime** [G-U-N/AnimateLCM#24]
[Here in an example inference config](https://github.com/G-U-N/AnimateLCM/blob/180828447a48ec71afda6b582f4d5b24104e757b/animatelcm_sd15/configs/inference-t2v.yaml#L29C19-L29C25) you use "linear" for inference, while the SD1.5 uses "scaled_linear", it seems to make quality much worse. Is this an intentional choice or an error?

**Comment by G-U-N** [comment]
Intentional choice. The model is trained with linear scheduler.

Here's an intuitive explanation: Generally say, we need to use a more noisy scheduler for images with  larger resolution, see [paper](https://arxiv.org/abs/2301.10972). Videos can be perceived as temporal stacks of images, so you can think its resolution is large. Linear scheduler is more noisy than scaled linear. Therefore I train the model with linear. 

Hope this clarifies any confusion.

**Comment by bonlime** [comment]
The explanation makes sense, but just from the personal experience using "linear" results in very over-saturated images, and therefor you have a hack with applying lora with 0.8 weight. If instead you switch to "scaled_linear" the images are less saturated buy also tend to have more details, and in combination with higher CFG result in much better videos overall

**Comment by G-U-N** [comment]
Not really sure about that.  Did you do any experiments on that? I think after training, the model will adapt to be good as long as the scheduler belonging to a reasonable interval. 

If you use the real image/video to supervise it, it should be able to reflect the property of your training data no matter what your scheduler is. 

That is different from simply changing "scale linear" to "linear" without any finetuing.

**Comment by bonlime** [comment]
I think the reason why it still works better with "scaled_linear" is due to the fact the the base SD1.5 used that schedule and your fine-tuning still can't change the model enough to adapt to "linear" schedule. You can just try and experiment with it a little bit yourself, to be the difference is very clear (using sd1.5 + T2V)

**Comment by G-U-N** [comment]
Any examples?
